pydantic_ai_examples/chat_app.py

```diff
@@ -136,9 +136,10 @@ async def stream_messages():
     return StreamingResponse(stream_messages(), media_type='text/plain')
 
 
-MessageTypeAdapter: TypeAdapter[ModelMessage] = TypeAdapter(
-    Annotated[ModelMessage, Field(discriminator='message_kind')]
-)
+# MessageTypeAdapter: TypeAdapter[ModelMessage] = TypeAdapter(
+#     Annotated[ModelMessage, Field(discriminator='message_kind')]
+# )
```

Comment:
i think this is a mistake, and should be uncommented, also this is where the conflicts are coming from.